docs/src/replica_cluster.md

```diff
@@ -1,102 +1,382 @@
+    - **Snapshot/Object Store Bootstrap**: Use the `recovery` section to
+      bootstrap from a volume snapshot or an object store.
+3. **Continuous Recovery Strategy**: Define this in the `.spec.replica` stanza:
+    - **Distributed Topology**: Configure using the `primary`, `source`, and
+      `self` fields along with the distributed topology defined in
+      `externalClusters`. This allows CloudNativePG to declaratively control the
+      demotion of a primary cluster and the subsequent promotion of a replica cluster
+      using a promotion token.
+    - **Standalone Replica Cluster**: Enable continuous recovery using the
+      `enabled` option and set the `source` field to point to an
+      `externalClusters` name. This configuration is suitable for creating replicas
+      primarily intended for read-only workloads.
+
+Both the Distributed Topology and the Standalone Replica Cluster strategies for
+continuous recovery are thoroughly explained below.
+
+## Distributed Topology
+
+!!! Important
+    The Distributed Topology strategy was introduced in CloudNativePG 1.24.
+
+### Planning for a Distributed PostgreSQL Database
+
+As Winston Churchill famously said, "Planning is essential", and this holds
+true for designing PostgreSQL architectures in Kubernetes.
+
+First, conceptualize your distributed topology on paper, and then translate it
+into a CloudNativePG API configuration. This configuration primarily involves:
+
+- The `externalClusters` section, which must be included in every `Cluster`
+  definition within your distributed PostgreSQL setup.
+- The `.spec.replica` stanza, specifically the `primary`, `source`, and
+  (optionally) `self` fields.
+
+For example, suppose you want to deploy a PostgreSQL cluster distributed across
+two Kubernetes clusters located in Southern Europe and Central Europe.
+
+In this scenario, assume you have CloudNativePG installed in the Southern
+Europe Kubernetes cluster, with a PostgreSQL `Cluster` named `cluster-eu-south`
+acting as the primary. This cluster has continuous backup configured with a
+local object store. This object store is also accessible by the PostgreSQL
+`Cluster` named `cluster-eu-central`, installed in the Central European
+Kubernetes cluster. Initially, `cluster-eu-central` functions as a replica
+cluster. Following a symmetric approach, it also has a local object store for
+continuous backup, which needs to be read by `cluster-eu-south`. The recovery
+in this setup relies solely on WAL shipping, with no streaming connection
+between the two clusters.
+
+Here’s how you would configure the `externalClusters` section for both
+`Cluster` resources:
+
+```yaml
+# Distributed topology configuration
+externalClusters:
+  - name: cluster-eu-south
+    barmanObjectStore:
+      destinationPath: s3://cluster-eu-central/
+      # Additional configuration
+  - name: cluster-eu-central
+    barmanObjectStore:
+      destinationPath: s3://cluster-eu-south/
+      # Additional configuration
```

Comment:
My brain is twisted at this point.
The `south` cluster will use the `cluster-eu-central` as source, but the storage for `cluster-eu-central` is in the south.
So, we do a crossing here, and then we do another crossing in the replica section below.
But, the storage for the `south` cluster is likely in the south, no?